README

```diff
@@ -8,3 +10,18 @@ To build the ding-libs from the tarball:
 To build the ding-libs from a source checkout:
     autoreconf -if
 then follow the tarball instructions above.
+
+# Requirements
+
+Build:
+
+- `autotools`
+- `gcc`
+- `libc`
+- `coreutils`
+
+Tests:
+
+- `file`
```

Comment:
Where is `file` used?

Reply:
Ah, I added `check` and `file` at the same time but it might not be strictly needed:

`./configure: line 7064: /usr/bin/file: not found`

Reply:
Then this is build time dep.

Reply:
You also check for `check` in `configure` :) But noted.

Reply:
And it compiles without it so some might argue it's optional.